spec.html

```diff
@@ -25642,11 +25642,11 @@
 
       <!-- es6num="20.3.3.4" -->
       <emu-clause id="sec-date.utc">
-        <h1>Date.UTC ( _year_, _month_ [ , _date_ [ , _hours_ [ , _minutes_ [ , _seconds_ [ , _ms_ ] ] ] ] ] )</h1>
-        <p>When the `UTC` function is called with fewer than two arguments, the behaviour is implementation-dependent. When the `UTC` function is called with two to seven arguments, the following steps are taken:</p>
+        <h1>Date.UTC ( _year_ [ , _month_ [ , _date_ [ , _hours_ [ , _minutes_ [ , _seconds_ [ , _ms_ ] ] ] ] ] ] )</h1>
+        <p>When the `UTC` function is called, the following steps are taken:</p>
         <emu-alg>
           1. Let _y_ be ? ToNumber(_year_).
-          1. Let _m_ be ? ToNumber(_month_).
+          1. If _month_ is supplied, let _m_ be ? ToNumber(_month_); else let _m_ be 0.
```

Comment:
@bterlson I just want to confirm this for clarification: we'll have a different behavior for supplied but undefined month. Like the following expected values:

```js
Date.UTC(2016) // 1451606400000
Date.UTC(2016, 0) // 1451606400000
Date.UTC(2016, undefined) // NaN
```

This will be consistent with the other parameters, but `Date.UTC` will never be perfect.